fili-core/src/main/java/com/yahoo/bard/webservice/data/metric/LogicalMetricInfo.java

```diff
@@ -82,6 +84,11 @@ public LogicalMetricInfo(String name, String longName, String description) {
         this(name, longName, LogicalMetric.DEFAULT_CATEGORY, description, TYPE_DEFAULT);
     }
 
+    @Override
+    public String getId() {
+        return getName();
```

Comment:
Since `getName` and `getDescription` are now overriding methods on an interface, we should annotate them with `@Override`.